chain-tx-enclave-next/mls/src/group.rs

```diff
@@ -204,20 +211,23 @@ impl GroupAux {
 
     fn get_welcome_msg(
         &self,
-        updated_tree: &TreePublicKey,
+        updated_tree: TreePublicKey,
         updated_group_context: &GroupContext,
         updated_secrets: &EpochSecrets<Sha256>,
         confirmation: Vec<u8>,
         interim_transcript_hash: Vec<u8>,
         positions: Vec<(NodeSize, KeyPackage)>,
     ) -> Result<Welcome, CommitError> {
+        let tree_hash = updated_tree.compute_tree_hash();
```

Comment:
I only see this computed, but does it get verified in `integrity_check`?
> Verify that the tree hash of the ratchet tree matches the tree_hash field in the GroupInfo.

Reply:
it's now verified in `init_group_from_welcome`.